### ISSUE TITLE
azurerm_app_service_virtual_network_swift_connection panic if app service is connected to storage account

### ISSUE THREAD

**Issue by abydal** [terraform-providers/terraform-provider-azurerm#11478]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Terraform (and AzureRM Provider) Version
```bash
Terraform v0.15.0
on windows_amd64
+ provider registry.terraform.io/hashicorp/azuread v1.4.0
+ provider registry.terraform.io/hashicorp/azurerm v2.56.0
+ provider registry.terraform.io/hashicorp/random v3.1.0
```

### Affected Resource(s)

* `azurerm_app_service_virtual_network_swift_connection`

### Terraform Configuration Files

https://gist.github.com/abydal/5b15c8a3f854483434ad08c166b23fe2

### Debug Output

https://gist.github.com/abydal/a6ebea2579372e98668f2df98b8dec10

### Panic Output

https://gist.github.com/abydal/a6ebea2579372e98668f2df98b8dec10

### Expected Behaviour

A resource azurerm_app_service_virtual_network_swift_connection  should be created

### Actual Behaviour

panic: value is marked, so must be unmarked first

### Steps to Reproduce

Try to create a `azurerm_app_service_virtual_network_swift_connection`  connected to an `app_service`  which also is connected to a 
`storage_account` resource. 

The `azurerm_app_service_virtual_network_swift_connection` example in the azurerm documentation fails if you add a storage account to the `app_service`. Unless you mark the `app_service.storage_account.access_key` as nonsensitive. (see submitted config file gist) 

### Important Factoids

This is not necessarily a bug, but the error combined with no documentation about this makes it really hard to understand why it fails.

**Comment by aristosvo** [comment]
Sounds like [this](https://github.com/hashicorp/terraform/pull/28383) Terraform 0.15 bug to me, which should be fixed in 0.15.1

**Comment by abydal** [comment]
I can confirm that an upgrade to Terraform 0.15.1 resolves this issue. It is no longer necessary to mark the `app_service.storage_account.access_key` as nonsentive in order to avoid the crash.
